cf-231A.mdx

```diff
@@ -0,0 +1,176 @@
+Once the outer for loop is done iteration, we can just print the main counter. 
+Below are the CPP and Java implementations of this solution. 
+
+There is another faster and simpler approach to this problem. Before you read the paragraph below this, 
+think about what you can do to make this problem faster. What mathmatical properties were discussed in 
+the previous solution that we can exploit? If you have a solid idea or are stuck, continue reading.
+
+The much simpler but more logic based solution would be to creat a for loop to iterate through n problems, 
+input all 3 numbers as integers and write an if statement to check if the sum of the integers is greater than 
+or equal to 2, and if it is, adding one to a main counter which we will then print out at the end, 
+similar to the last approach. Below are the CPP and Java implementations of this solution. 
+
+This is an example of using math to simplify and create an efficient program using math for greedy algorithms. 
+Next time you ever think of an approach to solve a problem, consider any alternate simpler and faster algorithms
+you can create. 
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+	
+***Method One***
+	
+```CPP
+#include <iostream>
+using namespace std;
+
+int main() {
+	int num = 0;
+	int n; cin >> n;
+	int problems[n][3];
+	for(int i = 0; i < n; i++){
+		cin >> problems[i][0];
+		cin >> problems[i][1];
+		cin >> problems[i][2];
+	}
+	for(int i = 0; i < n; i++){
+		int temp = 0;
+		for(int j = 0; j < 3; j++){
+			if(problems[i][j] == 1){
+				temp++;
+			}
+		}
+		if(temp >= 2){
+			num++;
+		}
+	}
+	cout << num << '\n';
+} 
+```
+
+***Method Two***
+	
+```CPP
+#include <iostream>
+using namespace std;
+
+int main() {
+	int num = 0;
+	int n; cin >> n;
+	for(int i = 0; i < n; i++){
+		int a; int b; int c; cin >> a; cin >> b; cin >> c;
+		if(a + b + c >= 2){
+			num++;
+		}
+	}
+	cout << num << '\n';
+} 
+```
+
+</CPPSection>
+	
+<JavaSection>
+	
+***Method One***
+
+```java
+import java.util.*;
+import java.io.*;
+	
+public class Main {
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+
+        	int n = io.nextInt();
+        	int[][] problems = new int[n][3];
+        
+        	for (int i = 0; i < n; i++) {
+            		problems[i][0] = io.nextInt();
+            		problems[i][1] = io.nextInt();
+            		problems[i][2] = io.nextInt();
+        	}
+
+        	int num = 0;
+
+        	for(int i = 0; i < n; i++) {
+            		int temp = 0;
+            		for(int j = 0; j < 3; j++) {
+                		if(problems[i][j] == 1){
+                   			temp++;
+                		}
+            		}
+            		if(temp >= 2){
+                		num++;
+            		}
+        	}
+     		System.out.println(num);
+        	io.close();                  
+	}
+	//CodeSnip{Kattio}
+}
+
+```
+***Method Two***
+	
+```java
+import java.util.*;
+import java.io.*;
+	public static void main(String[] args) {
```

Comment:
did you computer glitch out while copy pasting this

Reply:
whats wrong with it